src/main/java/ch/kalunight/zoe/repositories/PlayerRepository.java

```diff
@@ -0,0 +1,162 @@
+    }
+  }
+
+  public static void updateTeamOfPlayer(long playerId, long teamId) throws SQLException {
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement();) {
+
+      String finalQuery = String.format(UPDATE_TEAM_OF_PLAYER, teamId, playerId);
+      query.executeUpdate(finalQuery);
+    }
+  }
+
+  public static void updateTeamOfPlayerDefineNull(long playerId) throws SQLException {
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement();) {
+
+      String finalQuery = String.format(UPDATE_TEAM_OF_PLAYER_DEFINE_NULL, "NULL", playerId);
+      query.executeUpdate(finalQuery);
+    }
+  }
+
+  public static void deletePlayer(long playerId, long guildId) throws SQLException {
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement();) {
+
+      List<DTO.LeagueAccount> leaguesAccounts = LeagueAccountRepository.getLeaguesAccountsWithPlayerID(guildId, playerId);
+      
+      for(DTO.LeagueAccount leagueAccount : leaguesAccounts) {
+        LeagueAccountRepository.deleteAccountWithId(leagueAccount.leagueAccount_id);
+      }
+      
+      String finalQuery = String.format(DELETE_PLAYER_WITH_PLAYER_ID, playerId);
+      query.executeUpdate(finalQuery);
+    }
+  }
+
+  public static List<DTO.Player> getPlayers(long guildId) throws SQLException {
+    ResultSet result = null;
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);) {
+
+      String finalQuery = String.format(SELECT_PLAYERS_WITH_GUILD_ID, guildId);
+      result = query.executeQuery(finalQuery);
+
+      List<DTO.Player> accounts = new ArrayList<>();
+      if(0 != (result.last() ? result.getRow() : 0)) {
+        result.first();
+        while(!result.isAfterLast()) {
+          accounts.add(new DTO.Player(result));
+          result.next();
+        }
+      }
+
+      return accounts;
+    }finally {
+      RepoRessources.closeResultSet(result);
+    }
+  }
+
+  public static DTO.Player getPlayer(long guildId, long playerDiscordId) throws SQLException {
+    ResultSet result = null;
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);) {
+
+      String finalQuery = String.format(SELECT_PLAYER_WITH_DISCORD_ID, guildId, playerDiscordId);
+      result = query.executeQuery(finalQuery);
+      int rowCount = result.last() ? result.getRow() : 0;
+      if(rowCount == 0) {
+        return null;
+      }
+      return new DTO.Player(result);
+    }finally {
+      RepoRessources.closeResultSet(result);
+    }
+  }
+
+  @Nullable
+  public static DTO.Player getPlayerByLeagueAccountAndGuild(long guildId, String summonerId, String server) throws SQLException {
+    ResultSet result = null;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)